_releases/release-60400.md

```diff
@@ -18,8 +18,7 @@ The ROOT team is proud to announce the release of ROOT 6.04/00 featuring many us
 *   a new `TFormula` implementation that uses cling
 *   I/O of C++11 containers
 *   TTreeCache is now enabled by default to accelerate analyses
-*   many new beautiful palettes ([kBird!](https://root.cern/doc/master/classTColor.html#C06)) that are optimized against
-[monotonic perceptual ordering]( {% post_url 2014-05-31-rainbow-color-map %} )
+*   many new beautiful palettes ([kBird!](https://root.cern/doc/master/classTColor.html#C06)) that are optimized against monotonic perceptual ordering
```

Comment:
this content actually exists! when the site is generated, it is as `/baseurl/rainbow-color-map`

Reply:
aaah it's a blog post!, i'll try a relative url

Reply:
i'm too fast in deleting links forever :sweat_smile:

Reply:
Done!